mesos/jenkins/jenkins.sh

```diff
@@ -25,16 +25,30 @@ copy_reference_file() {
 : ${JENKINS_HOME:="/var/jenkins_home"}
 export -f copy_reference_file
 touch "${COPY_REFERENCE_FILE_LOG}" || (echo "Can not write to ${COPY_REFERENCE_FILE_LOG}. Wrong volume permissions?" && exit 1)
-echo "--- Copying files at $(date)" >> "$COPY_REFERENCE_FILE_LOG"
-find /usr/share/jenkins/ref/ -type f -exec bash -c "copy_reference_file '{}'" \;
 
-sed -i "s#\${JENKINS_CONFIG_REPO}#$JENKINS_CONFIG_REPO#g" $JENKINS_HOME/scm-sync-configuration.xml
-cp -R $MESOS_SANDBOX/.ssh $JENKINS_HOME/.ssh
+if [ "x$USE_PERSISTANT_JENKINS_HOME" == "x" ]; then
+	rm -f $JENKINS_HOME/.USE_PERSISTANT_JENKINS_HOME
+	echo "--- Copying files at $(date)" >> "$COPY_REFERENCE_FILE_LOG"
+	find /usr/share/jenkins/ref/ -type f -exec bash -c "copy_reference_file '{}'" \;
+	sed -i "s#\${JENKINS_CONFIG_REPO}#$JENKINS_CONFIG_REPO#g" $JENKINS_HOME/scm-sync-configuration.xml
+	cp -R $MESOS_SANDBOX/.ssh $JENKINS_HOME/.ssh
+else
+	if [ ! -f $JENKINS_HOME/.USE_PERSISTANT_JENKINS_HOME ]; then
```

Comment:
This shouldn't be cut-and-paste but a function that can be called. The if/else is a bit confusing as well. Why not use `elif`?